src/index.html

```diff
@@ -310,6 +331,14 @@ <h3>This html is duplicated in <a>https://bitbucket.org/skyslope/skyslope-widget
             <p>We also have single widget that can easily be dropped anywhere</p>
             <ss-contained-widget class="contained-widget"></ss-contained-widget>
           </div>
+          <div class="card">
+            <p>Enter a valid envelope id to open the widget in the form filling experience.<br>
+              <i>Note: Envelope must be in draft status (not sent, completed or canceled).</i></p>
+            <div>
+              <button onclick="openEnvelope()">Go to Envelope</button>
+              <input id="goto-envelope-id" class="envelope-id-input" inputmode="numeric" oninput="this.value = this.value.replace(/\D+/g, '')" maxlength="12" />
```

Comment:
Huh, I've never seen `oninput` before! Would it be simpler and potentially accomplish the same thing to use `type="number"` on the input?